prep-css-box-model/style.css

```diff
@@ -0,0 +1,14 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+  margin: 20px;
+  padding: 10px;
+  border-style: double;
+  border: darkgray;
```

Comment:
Here you will want to specify the width of the border in `px` as well as the color so that the border can be seen.